app/models/internal_change_note.rb

```diff
@@ -3,6 +3,6 @@ class InternalChangeNote < ApplicationRecord
   belongs_to :step_by_step_page
 
   def readable_created_date
-    created_at.strftime('%A, %d %B %Y at %H:%M %p')
+    created_at.strftime('%A, %d %B %Y at %I:%M %p')
```

Comment:
What's the difference between `%H` and `%I`?

Reply:
H is 24 hours . so 4pm will be 16

Reply:
Didn't changing this break any tests?

Reply:
Can you add a note about this in the commit message?